src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -683,4 +704,149 @@ public function column_cb( $post ) {
 			<div class="locked-indicator"></div>
 		<?php
 	}
+
+	/**
+	 * Displays the pagination.
+	 *
+	 * @since 5.2.2
+	 * @access protected
+	 *
+	 * @param string $which
```

Comment:
:no_entry_sign: Missing parameter comment

<sub>hash: bcca1f04d48de41be19177d9d7b8800d</sub>

Reply:
Make sure to resolve this issue as well